test_data/makedata_suites/610_analyzers.js

```diff
@@ -0,0 +1,255 @@
+      };
+      let classifierDoubleExpectedResult =[ 
+        { 
+          "double" : [ 
+            "__label__baking", 
+            "__label__bread" 
+          ] 
+        } 
+      ];
+
+      let classifierDoubleQueryReuslt = db._query(`LET str = "Which baking dish is best to bake a banana bread ?" RETURN {"double": TOKENS(str, "${classifierDouble}")}`);
+
+      checkAnalyzer(classifierDouble, classifierDoubleType, classifierDoubleProperties, classifierDoubleExpectedResult, classifierDoubleQueryReuslt)
+
+      //-------------------------------nearestNeighborsSingle----------------------------------
+
+      let nearestNeighborsSingle = `nearestNeighborsSingle_${dbCount}`;
+      let nearestNeighborsSingleType = "nearest_neighbors";
+      let nearestNeighborsSingleProperties = { 
+        "model_location" : `${path}`, 
+        "top_k" : 1 
+      };
+      let nearestNeighborsSingleExpectedResult =[ 
+        { 
+          "all" : [ 
+            "ingredients", 
+            "as" 
+          ] 
+        } 
+      ];
+
+      let nearestNeighborsSingleQueryReuslt = db._query(`LET str = "salt, oil"RETURN {"all": TOKENS(str, "${nearestNeighborsSingle}")}`);
+
+      checkAnalyzer(nearestNeighborsSingle, nearestNeighborsSingleType, nearestNeighborsSingleProperties, nearestNeighborsSingleExpectedResult, nearestNeighborsSingleQueryReuslt)
+
+      //-------------------------------nearestNeighborsDouble----------------------------------
+
+      let nearestNeighborsDouble = `nearestNeighborsDouble_${dbCount}`;
+      let nearestNeighborsDoubleType = "nearest_neighbors";
+      let nearestNeighborsDoubleProperties = { 
+        "model_location" : `${path}`, 
+        "top_k" : 2 
+      };
+      let nearestNeighborsDoubleExpectedResult =[ 
+        { 
+          "double" : [ 
+            "ingredients", 
+            "whole", 
+            "as", 
+            "in" 
+          ] 
+        }  
+      ];
+
+      let nearestNeighborsDoubleQueryReuslt = db._query(`LET str = "salt, oil"RETURN {"double": TOKENS(str, "${nearestNeighborsDouble}")}`);
+
+      checkAnalyzer(nearestNeighborsDouble, nearestNeighborsDoubleType, nearestNeighborsDoubleProperties, nearestNeighborsDoubleExpectedResult, nearestNeighborsDoubleQueryReuslt)
+
+
+      return 0;
+
+    },
+    clearDataDB: function (options, isCluster, isEnterprise, dbCount, database) {
+      print(`checking data ${dbCount}`);
+      // deleting analyzer
+      function deleteAnalyzer(analyzerName){
+        try {
+          const array = a.toArray();
+          for (let i = 0; i < array.length; i++) {
+            const name = array[i];
+            if (name === analyzerName) {
+              a.remove(analyzerName);
+            }
+          }
+          // checking created text analyzer is deleted or not
+          if (a.analyzer(analyzerName) != null) {
+            throw new Error(`${analyzerName} analyzer isn't deleted yet!`);
+          }
+        } catch (e) {
+          print(e);
```

Comment:
Is this print required? Or should it throw instead?

Reply:
Here we tried to notify without throwing, then cleanup should do its best to clean the system.